### PR TITLE
 Once modules have been uploaded into Pantheon using the dynamic incl…

### PR DESCRIPTION
**Pull request by rednitish**
…ude syntax, Pantheon needs to populate the attsFile attribute similar to the following CLI invocation of asciidoctor

### CODE REVIEW

**Comment by benradey** [comment]
Hey @rednitish , thanks for opening this PR. For future PRs, can you please request review from @redhataccess/pantheon-developers ? For this one, I already assigned it to myself, so no need to do it now. But that way, we all get notified when there's something new to review. Thanks!

**Comment by rednitish** [comment]
Yes, sure. Thanks for letting me know Ben..😊

File: pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java [hunk 1]
```diff
@@ -179,10 +179,15 @@ private String buildModule(Module base, ModuleVersion moduleVersion, Map<String,
                     .map(Supplier::get);
 
             SimpleDateFormat dateFormat = new SimpleDateFormat("dd MMMMM yyyy");
+
+            String workspacePath = moduleVersion.getWorkspace().getPath();
+            String attributeFileRelPath =  moduleVersion.getWorkspace().attributeFile().get();
```
Comment (benradey) [inline pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java:184]: Small nitpick, but could you remove the double-spacing after the equal sign?

`...elPath =  modu...` <-- double-spacing here.

File: pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java [hunk 2]
```diff
@@ -179,10 +179,15 @@ private String buildModule(Module base, ModuleVersion moduleVersion, Map<String,
...
             // build the attributes (default + those coming from http parameters)
             AttributesBuilder atts = AttributesBuilder.attributes()
                     // show the title on the generated html
                     .attribute("showtitle")
+                    // provide attribute file as argument to ASCIIDOCTOR for building doc.
+                    .attribute("attsFile", workspacePath.concat( "/"+ attributeFileRelPath))
```
Comment (benradey) [inline pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java:190]: Two comments here... first of all, please fix the spacing in this code. :) 

`...concat( "/"+ attribute...`

File: pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java [hunk 3]
```diff
@@ -240,14 +245,10 @@ private String buildModule(Module base, ModuleVersion moduleVersion, Map<String,
...
 
-                String workspacePath = moduleVersion.getWorkspace().getPath();
-                String attributeFileRelPath =  moduleVersion.getWorkspace().attributeFile().get();
                 StringBuilder content = new StringBuilder();
                 if (attributeFileRelPath != null && !attributeFileRelPath.isEmpty()) {
                     content.append("include::")
-                            .append(workspacePath)
-                            .append("/")
-                            .append(attributeFileRelPath)
+                            .append("{attsFile}")
```
Comment (benradey) [inline pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java:251]: Clever! I wouldn't have thought of this, but I like the logic!

**Comment by rednitish** [comment]
Changed the code as per review received, can you please review and approve

File: pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java [hunk 4]
```diff
@@ -179,10 +179,15 @@ private String buildModule(Module base, ModuleVersion moduleVersion, Map<String,
...
             SimpleDateFormat dateFormat = new SimpleDateFormat("dd MMMMM yyyy");
+
+            String workspacePath = moduleVersion.getWorkspace().getPath();
+            String attributeFileRelPath = moduleVersion.getWorkspace().attributeFile().get();
             // build the attributes (default + those coming from http parameters)
             AttributesBuilder atts = AttributesBuilder.attributes()
                     // show the title on the generated html
                     .attribute("showtitle")
+                    // provide attribute file as argument to ASCIIDOCTOR for building doc.
+                    .attribute("attsFile", workspacePath + "/"+ attributeFileRelPath)
```
Comment (benradey) [inline pantheon-bundle/src/main/java/com/redhat/pantheon/asciidoctor/AsciidoctorService.java:190]: You're missing a space after the "/", but I won't ask for another commit, just please keep an eye out in the future. :)

Thanks!

**Pull request merged**